src/rock/models/content/model.js

```diff
@@ -0,0 +1,90 @@
+import { merge, isNil, pick, flatten, find, uniq, sampleSize } from "lodash";
+import { defaultCache } from "../../../util/cache";
+import { createGlobalId } from "../../../util";
+
+import { ContentChannel, ContentChannelItem } from "./tables";
+import { AttributeValue, Attribute } from "../system/tables";
+
+import { Rock } from "../system";
+
+export class RockContent extends Rock {
+  __type = "RockContent";
+
+  constructor({ cache } = { cache: defaultCache }) {
+    super();
+    this.cache = cache;
+  }
+
+  async getAttributeFromId(key, id, globalId) {
```

Comment:
I don't think we will need this. It can be moved into the find query rather than making a separate function.